pkgs/tools/misc/wayback/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, pkg-config
+, libtool
+, chromium
+}:
+
+buildGoModule rec {
+  pname = "wayback";
+  version = "0.19.1";
+
+  src = fetchFromGitHub {
+    owner = "wabarc";
+    repo = "wayback";
+    rev = "v${version}";
+    hash = "sha256-LIWCT0/5T52VQQK4Dy6EFmFlJ02MkfvKddN/O/5zpZc=";
+  };
+
+  vendorHash = "sha256-TC4uwJswpD5oKqF/rpXqU/h+k0jErwhguT/LkdBA83Y=";
+
+  doCheck = false;
+
+  buildInputs = [
+    pkg-config
+    libtool
+  ];
```

Comment:
I am somewhat sure those are also nativeBuildInputs

Reply:
Would putting those both in buildInputs and nativeBuildInputs be fine?

Reply:
Why do they need to be in both? Aren't they used at build time only?

Reply:
Yes. Only chromium is the runtime dependency.

Reply:
Then we shouldn't require them in both.

Reply:
It turned out it can be built without any of them, sorry.